guide/src/web-getting-started.md

```diff
@@ -43,35 +43,44 @@ You can import the components just like other things you’re used to in JavaScr
 import { FerrostarMap, BrowserLocationProvider } from "@stadiamaps/ferrostar-components";
 ```
 
-## Configure the `<ferrostar-map>` component
+## Configure the <ferrostar-map> Component
 
-Now you can use Ferrostar in your HTML like this:
+Now you can use Ferrostar in your React component like this:
 
-```html
+```javascript
+import React from 'react';
+import { FerrostarMap } from "@stadiamaps/ferrostar-webcomponents";
+
+const MyMapComponent = () => {
+return (
 <ferrostar-map
-  id="ferrostar"
-  valhallaEndpointUrl="https://api.stadiamaps.com/route/v1"
-  styleUrl="https://tiles.stadiamaps.com/styles/outdoors.json"
-  profile="bicycle"
-></ferrostar-map>
+id="ferrostar"
+valhallaEndpointUrl="https://api.stadiamaps.com/route/v1"
+styleUrl="https://tiles.stadiamaps.com/styles/outdoors.json"
+profile="bicycle"
+style={{ display: 'block', width: '100%', height: '100%' }} ></ferrostar-map>
+);
+};
+
+export default MyMapComponent;
 ```
```

Comment:
Why delete the HTML example? This still works great :) We can add details about react but not delete the original.

Reply:
Sorry for this change, as i said i am going to undo this change and make sure that this should be along with React and not  to replace the HTML example.

Is it fine now like this :
--

Now you can use Ferrostar in your Web component like this:

## HTML

```html
<ferrostar-map
id="ferrostar"
  valhallaEndpointUrl="https://api.stadiamaps.com/route/v1"
  styleUrl="https://tiles.stadiamaps.com/styles/outdoors.json"
  profile="bicycle"
></ferrostar-map>
```

## React

```javascript
import React from 'react';
import { FerrostarMap } from "@stadiamaps/ferrostar-webcomponents";

const MyMapComponent = () => {
return (
<ferrostar-map
id="ferrostar"
valhallaEndpointUrl="https://api.stadiamaps.com/route/v1"
styleUrl="https://tiles.stadiamaps.com/styles/outdoors.json"
profile="bicycle"
style={{ display: 'block', width: '100%', height: '100%' }} ></ferrostar-map>
);
};

export default MyMapComponent;
```